File: ProjectFindDatabase/Models/Students.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjectFindDatabase.Models
{
    public class Students
    {
        public int Id { get; set; }
        public string Username { get; set; }
        public DateTime StartDateFax { get; set; }
        public int Faculty_Id { get; set; }
        public bool Academy { get; set; }
        public string AcademyName { get; set; }
        public int GradeLevel { get; set; }
        public Helpers.Enumerations.Degree Degree { get; set; }
        public string StudentEmail { get; set; }
        public int Skill_Id { get; set; }
        public string Biography { get; set; }
    }
}

File: ProjectFindWEB/ProjectFind/src/app/models/students.ts
import { Degree } from '../enums/degree.enum';

export class Students {
	id: number;
	username: string;
	startDateFax: Date;
	faculty_Id: number;
	academy: boolean;
	academyName: string;
	gradeLevel:number;
	degree: Degree;
	studentEmail: string;
	skill_Id: number;
	biography: string;
	consttructor(
		id?: number,
		username?: string,
		startDateFax?: Date,
		faculty_Id?:number,
		academy?: boolean,
		academyName?: string,
		gradeLevel?:number,
	    degree?: Degree,
		studentEmail?: string,
		skill_Id?: number,
		biography?: string
	){
		this.id=id;
		this.username=username;
		this.startDateFax = startDateFax;
		this.faculty_Id = faculty_Id;
		this.academy = academy;
		this.academyName = academyName;
		this.gradeLevel=gradeLevel;
		this.degree=degree;
		this.studentEmail=studentEmail;
		this.skill_Id=skill_Id;
		this.biography=biography;
	}
}
